### PR TITLE
cli plus instructions

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@aviflombaum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,93 @@
...
-languages: 
-resources: 
----
-# Ttt 5 Move Rb
-
-*Idea: Give them #display_board so they can see output*
\ No newline at end of file
+# Tic Tac Toe CLI: Adding Player Move to the Game Board
+
+## Objectives
```
Comment (aviflombaum) [inline README.md:3]: These feel more like instructions. The Objectives are really:

define a method that updates an array passed to it.
define a method with a default value
use a method in a CLI
accept user input via gets
use user input in a method.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+
+**Hint:** Your `#move` method must take in three arguments, the board array, the location in the board array that the player would like to fill out with an "X" or and "O", and the player's character (either "X" or "O"). If the user's input is `5`, the player wants to fill out position 5 with their character. This means that your method must fill out the correct array index with the player's character. Remember that, from the player's point of view, the board contains spaces 1-9. But, an array's indexes start their count at 0. You'll have to account for that in your `#move` method. 
+
+Once you have the tests passing, move on to part II. 
+
+### Part II: The CLI
+
+Open up `bin/move`. We're ready to code the executable portion of this program. 
+
+1. Our program should first welcome the player by outputting a friendly message to the terminal: "Welcome to Tic Tac Toe".
```
Comment (aviflombaum) [inline README.md:88]: We should make sure to remind them that the CLI has to define the starting state, ie, defining an empty board to start.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,94 @@
...
+## Instructions
+
+### Part I: Defining our Methods
+
+The first part of this lab is test-driven. Run `learn` to get started and use the test output to guide you in defining the move method. 
+
+Notice that we've already given you the `#display_board` method, since we've already built that out in a previous exercise. 
+
+**Hint:** Your `#move` method must take in three arguments, the board array, the location in the board array that the player would like to fill out with an "X" or and "O", and the player's character (either "X" or "O"). If the user's input is `5`, the player wants to fill out position 5 with their character. This means that your method must fill out the correct array index with the player's character. Remember that, from the player's point of view, the board contains spaces 1-9. But, an array's indexes start their count at 0. You'll have to account for that in your `#move` method. 
+
```
Comment (aviflombaum) [inline README.md:81]: We should mention the default argument value requirement.

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md:3]: fixed w/ latest commit.

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md:88]: fixed w latest commit.

**Pull request merged**